pkg/api/controller_test.go

```diff
@@ -471,6 +474,12 @@ func TestController_LogCommitsPredefinedData(t *testing.T) {
 			expectedCommits: []string{"commit10", "commit1"},
 			expectedMore:    false,
 		},
+		{
+			name:            "stop_at",
+			expectedCommits: []string{"commit10", "commit9"},
+			expectedMore:    false,
+			stopAt:          commitsMap["commit9"].Reference,
```

Comment:
So stopAt is literally a ref, _not_ a commit ID.  I like this -- see above -- but we need to fix the OpenAPI docs if this is the way it should be.

```suggestion
			stopAt:          commitsMap[8].Reference,
```
(I think).

Also this might need to be a constant, given that totalCommits is also one.